### ISSUE TITLE
MySQL Workbench crashes

### ISSUE THREAD

**Issue by larrybreyer** [Homebrew/homebrew-cask#140670]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
% brew config
HOMEBREW_VERSION: 3.6.20-232-g39e158f
ORIGIN: https://github.com/Homebrew/brew
HEAD: 39e158fb939dadcdf6ee0a67a0930b15f9b16b39
Last commit: 2 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a3d6d076ea120c0a552589bcde39883a7c6a9d10
Core tap last commit: 2 days ago
Core tap branch: master
Core tap JSON: 06 Feb 15:19
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.q2IJZK2muG/org.xquartz:0
HOMEBREW_EDITOR: vim
HOMEBREW_INSTALL_FROM_API: set
HOMEBREW_MAKE_JOBS: 20
HOMEBREW_NO_INSTALL_CLEANUP: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 20-core 64-bit skylake
Clang: 14.0.0 build 1400
Git: 2.39.1 => /usr/local/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2

% brew doctor
Your system is ready to brew.
```

### Verification

- [X] My "`brew doctor` output" says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

I am running the MySQL Workbench (GUI).  Whenever I enter a query that requires output, it crashes.
Note: It connects to the database without any problems.  

I captured the latest crash report from Console.  It says,
Termination Reason:    Namespace SIGNAL, Code 4 Illegal instruction: 4

I uploaded the crash report via Gist to (https://gist.github.com/larrybreyer)
larrybreyer/[MySQLWorkbench.txt]

The logs in ~/Library/Application Support/MySQL/Workbench/log look good.
here is wb.log

[sql_actions_larry.log](https://github.com/Homebrew/homebrew-core/files/10670415/sql_actions_larry.log)

[wb.log](https://github.com/Homebrew/homebrew-core/files/10670413/wb.log)

### What happened (include all command output)?

When I enter the command:  "SHOW DATBASES:"  it crashes.
When I enter the command: "USE mysql;"  it works fine.
When I enter the command "SHOW TABLES;" it crashes.

### What did you expect to happen?

I expected the output window to show me the results or report an error.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
Here is the output from
"brew update && brew upgrade && brew install mysqlworkbench"

Already up-to-date.        
==> Downloading https://cdn.mysql.com/Downloads/MySQLGUITools/mysql-workbench-co
######################################################################## 100.0%
==> Installing Cask mysqlworkbench
==> Moving App 'MySQLWorkbench.app' to '/Applications/MySQLWorkbench.app'
🍺  mysqlworkbench was successfully installed!
```

**Comment by miccal** [comment]
Does this happen if you install `mysqlworkbench` manually?

**Comment by larrybreyer** [comment]
I have no idea what a manual install is.
Do you mean download directly from dev.mysql.com?

I am thinking, if I am having this issue, perhaps others are as well.
In that case I would think the solution is to fix the problem.
Are you saying the crash dump does not identify the problem?

--
Larry Breyer

**Comment by bevanjkay** [comment]
@larrybreyer Yes Miccal is referring to downloading and installing directly from dev.mysql.com.
If the issue exists when downloaded directly from the source and installed manually then unfortunately there is nothing we can do to help from our end, as we just provide systematic installations of the software. You would need to report the issue to MySQL.

**Comment by larrybreyer** [comment]
In order to install MySQL Workbench from dev.mysql.com should I first "brew
uninstall --cask mysqlworkbench"?

--
Larry Breyer
(858) 597-9880
(voice or text)

**Comment by SMillerDev** [comment]
Yes, you would

**Comment by larrybreyer** [comment]
Hi Guys,

I uninstalled the Homebrew version and installed
mysql-workbench-community-8.0.32-macos-x86_64.dmg from dev.mysql.com.

It was installed really easily.  It launches really easily.
But, it crashed just like the Homebrew version.

Do you have any recommendations on how I approach Oracle for help?

Thank you!

--
Larry Breyer

**Comment by bevanjkay** [comment]
@larrybreyer Thanks for confirming, and I hope you have success reporting the bug to MySQL. Here's a link that might help - https://bugs.mysql.com/
I'm going to go ahead and close this issue, due to it being a bug upstream.

**Comment by larrybreyer** [comment]
Hey Guys!

I uninstalled the Homebrew package and installed dev.mysql.com version.
I deleted the log files in ~/Library/Application
Support/MySQL/Workbench/log/
When I run the new package I find this in wb.log

17:07:35 [INF][SQL Editor Form]: Opened connection 'Local instance 3306' to
Homebrew version 8.0.32

Is it possible the package in dev.mysql.com was built by Homebrew?

--
Larry Breyer
(858) 597-9880
(voice or text)
